js/app.jsx

```diff
@@ -297,101 +416,94 @@ var CaseverListItem = React.createClass({
   }
 });
 
+var SearchTableHeadConstTh = React.createClass({
+  render: function(){
+    return(
+      <th>{this.props.text}</th>
+    )
+  }
+})
 
-var CaseverList = React.createClass({
+var SearchTableHeadTh = React.createClass({
+  handleSort: function(){
+    var newOrder = "";
+    if (this.state.order == "") {
+      newOrder = "-";
+    }
+    this.setState({"order": newOrder})
+    this.props.handleAddFilter(' orderby:' + newOrder + this.props.filter, / orderby:[-\w]+/)
+  },
+  getInitialState: function(){
```

Comment:
Yes. But based on our current API, we can only sort by one column. So it makes more sense to keep only one state in the `SearchableList`, like `{"sortby": "title", "order": "+"}`. It will also help with #84